pyspectrafuse/commands/build_cluster_db.py

```
@@ -0,0 +1,450 @@
+    if existing_metadata and existing_membership:
+        from pyspectrafuse.incremental.merge_results import merge_into_existing_db
+        # Merge into existing DB. Uses the FIRST parquet_dir/dataset_name as the
+        # "new data" source — callers invoking this mode are expected to pass
+        # exactly one new-data parquet_dir plus any number of rep scan_titles.
+        from pyspectrafuse.commands.spectrum2msp import find_target_ext_files
+        new_parquet_paths = []
+        for pdir in parquet_dir:
+            new_parquet_paths.extend(find_target_ext_files(pdir, '.parquet'))
+        if not new_parquet_paths:
+            raise click.ClickException(
+                f"No parquet files found in provided --parquet_dir entries")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)